code/game/objects/items/vending_items.dm

```diff
@@ -37,8 +37,8 @@
 /obj/item/vending_refill/boozeomat
 	machine_name = "Booze-O-Mat"
 	icon_state = "refill_booze"
-	charges = list(54, 4, 0)//of 159 standard, 12 contraband
-	init_charges = list(54, 4, 0)
+	charges = list(58, 4, 0)//of 162 standard, 12 contraband

```

Comment:
it's 174.

![image](https://user-images.githubusercontent.com/6209658/33808431-2c37dfc8-ddb4-11e7-9fdc-de1564ff76f5.png)